src/app/app-welcome/app-welcome.component.ts

```diff
@@ -13,12 +13,26 @@
 // limitations under the License.
 
 import {Component} from '@angular/core';
+
 import * as template from './app-welcome.content.html';
+import {LoginDialogComponent} from '../login-dialog/login-dialog.component';
+import {MatDialog, MatDialogConfig} from '@angular/material';
 
 @Component({
   selector: 'app-app-welcome',
   template: `<div class="container">${template}</div>`,
   styleUrls: ['./app-welcome.component.scss'],
 })
 export class AppWelcomeComponent {
+
+  constructor(private dialog: MatDialog) {}
```

Comment:
This is a duplication of https://github.com/DD-DeCaF/caffeine/pull/13/files#diff-773dfe9e1ea9e694466bd0d75d9d325cR41
Please merge them. You could maybe add it to a service, or create an angular component, which behaves like a link (with transclusion)